tests/components/unifi/test_switch.py

```diff
@@ -427,7 +428,10 @@ async def test_switches(hass, aioclient_mock):
     }
 
     await hass.services.async_call(
-        SWITCH_DOMAIN, "turn_on", {"entity_id": "switch.block_client_1"}, blocking=True
+        Platform.SWITCH,
```

Comment:
This API is domain-based, not platform-based. Thus should not be using the platform enum.

Many more cases like this are in this PR.